src/components/dashboard/HotspotCard.vue

```diff
@@ -180,11 +207,27 @@ const maxRace = computed(() => {
   return activeStats.value[0];
 });
 
+const maxInclAll = computed(() => {
+  for (const activeStat of activeStats.value) {
+    if (!isNaN(activeStat.rate)) {
```

Comment:
Are you looking for something like `isFinite()`?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isFinite